bin/demultiplexingTiming.sh

```diff
@@ -171,16 +173,17 @@ fi
 timeStampDir="${SCR_ROOT_DIR}/logs/Timestamp/"
 
 ## check if there are samplesheet available for run time checking.
-checkSampleSheet=$(find "${SCR_ROOT_DIR}/Samplesheets/" -maxdepth 1 -type f -name "*.csv")
+declare -a sampleSheets
+sampleSheets=$(find "${SCR_ROOT_DIR}/Samplesheets/" -maxdepth 1 -type f -name "*.csv")
 
-if [[ -z "${checkSampleSheet}" ]]
+if [[ -z "${sampleSheets}" ]]
```

Comment:
Je controleert hier of de string ${sampleSheets} leeg is, maar het is nu expliciet een array geworden ... Dat gaat niet goed. Tevens zal de combinatie van set -e + set -u maken dat het script al crasht als de waarde leeg is. Ik zou hier iets van:
```
if [[ "${#sampleSheets[@]:-0}" -eq '0' ]]
```
gebruiken; zie vergelijkbaar construct in bijvoorbeeld ```arrayConversion.sh```